Chummer/Backend/Uniques/CritterPower.cs

```diff
@@ -80,6 +81,11 @@ public void Create(XmlNode objXmlPowerNode, int intRating = 0, string strForcedV
             _nodBonus = objXmlPowerNode.SelectSingleNode("bonus");
             if (!objXmlPowerNode.TryGetMultiLineStringFieldQuickly("altnotes", ref _strNotes))
                 objXmlPowerNode.TryGetMultiLineStringFieldQuickly("notes", ref _strNotes);
+
+            String sNotesColor = ColorTranslator.ToHtml(ColorManager.HasNotesColor);
+            objXmlPowerNode.TryGetStringFieldQuickly("notesColor", ref sNotesColor);
+            _colNotes = ColorTranslator.FromHtml(sNotesColor);
+
```

Comment:
Is this necessary in the `Create` method? Would there need to be data-definable colors for specific critter powers?